src/Chapter2.hs

```diff
@@ -780,13 +790,13 @@ Let's now try to eta-reduce some of the functions and ensure that we
 mastered the skill of eta-reducing.
 -}
 divideTenBy :: Int -> Int
-divideTenBy x = div 10 x
+divideTenBy = div 10
 
--- TODO: type ;)
-listElementsLessThan x l = filter (< x) l
+listElementsLessThan :: Int -> [Int] -> [Int]
+listElementsLessThan = filter . (>)
```

Comment:
You do love the point-free style 😄 👍🏼

Reply:
Pointfree style is one of the few things that got me really interested in haskell tbh 😅